09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
-```
-
-This is of course a very simple example, and we could have achieved the same using `rgdal::ogrInfo()`.
-Still, it shows how we can use GDAL via the command-line from within R independently of other packages, and nothing stops us from using more advanced and/or not already implemented GDAL utilities in R.
+#> INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
+#>     using driver `ESRI Shapefile' successful.
+#> 
+#> Layer name: nc
+#> Metadata:
+#>  DBF_DATE_LAST_UPDATE=2016-10-26
+#> Geometry: Polygon
+#> Feature Count: 100
+#> Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
+#> Layer SRS WKT:
+#> GEOGCS["GCS_North_American_1927",
+#>     DATUM["North_American_Datum_1927",
+#>         SPHEROID["Clarke_1866",6378206.4,294.9786982]],
+#>     PRIMEM["Greenwich",0],
+#>     UNIT["Degree",0.017453292519943295],
+#>     AUTHORITY["EPSG","4267"]]
+#> AREA: Real (24.15)
+#> ...
+```
+
+This example --- which returns the same result as `rgdal::ogrInfo()` --- may be simple, but it shows how to use GDAL via the system command-line, independently of other packages.
+The 'link' to GDAL provided by **link2gi** could be used as a foundation for more doing more advanced GDAL work from the R or system CLI.^[
 Note also that the **RSAGA** package uses the command line interface to use SAGA geoalgorithms from within R (see section \@ref(rsaga)). 
-TauDEM (http://hydrology.usu.edu/taudem/taudem5/index.html) and the Orfeo Toolbox (https://www.orfeo-toolbox.org/) are further examples for offering a command line interface. 
+]
+TauDEM (http://hydrology.usu.edu/taudem/taudem5/index.html) and the Orfeo Toolbox (https://www.orfeo-toolbox.org/) are other low level spatial data processing libraries offering a command line interface. 
 At the time of writing it appears that there is only a developer version of an R/TauDEM interface on R-Forge (https://r-forge.r-project.org/R/?group_id=956)).
-In any case, you now have the tools to build an interface yourself.
+In any case, the above example shows how to access low level libraries from the system, creating the possibility for new R bridges to these libraries to be created.
 
-Please keep in mind that the **sf** package already brings most of the power provided by GDAL, GEOS and PROJ to R.
-Instead of using the command-line interface to access the functionality of these external dependencies, **sf** uses an **Rcpp** interface.
-This is an even better approach since it circumvents the command-line and allows a direct access to GDAL/GEOS/PROJ all of which are written in either C++ or C.
+Before diving into a project to create a new bridge, however, it is important to be aware of the power of existing R packages and that `system()` calls may not be platform-independent (they may fail on some computers).
+Furthermore, **sf** brings most of the power provided by GDAL, GEOS and PROJ to R via the R/C++ interface provided by **Rcpp**, which circumvents the command-line.
```

Comment:
well, I wouldn't say circumvent, they only use another API, i.e. the C++ API.

Reply:
Changed.